docker/validator/Dockerfile

```diff
@@ -16,7 +16,7 @@ FROM toolchain AS builder
 
 COPY . /libra
 
-RUN cargo build --release -p libra-node -p cli -p config-builder && cd target/release && rm -r build deps incremental
+RUN cargo build --release -p libra-node -p cli -p config-builder -p safety-rules && cd target/release && rm -r build deps incremental
```

Comment:
why add here too?

Reply:
This is to enable Docker layer caching so when CI builds all the images the build only happens once.